conditional/attendance/serializers.py

```diff
@@ -1,5 +1,5 @@
 from rest_framework import serializers
-from attendance.models import Directorship, DirectorshipMeeting, MemberDirectorshipAttendance
+from attendance.models import Directorship, DirectorshipMeeting, MemberDirectorshipAttendance, FreshmanDirectorshipAttendance, TechnicalSeminar, MemberSeminarAttendance, FreshmanSeminarAttendance
```

Comment:
I believe this can be reduced to a decorator. I'll do some playing around with it and if it works I'll toss you a commit.